bindings/python/pinocchio/visualize/base_visualizer.py

```diff
@@ -2,14 +2,35 @@
 from ..shortcuts import buildModelsFromUrdf, createDatas
 
 import time
+import numpy as np
+from pathlib import Path
```

Comment:
pathlib is in standard library only for python 3

Reply:
Ok, I fixed that and used `os.path` instead